### ISSUE TITLE
Error Code 535 PLEASE HELP!

### ISSUE THREAD

**Issue by TheRealMystical** [jef/streetmerchant#1191]
Error Code 535 PLEASE HELP!

I'm quite a novice at this and for both my phone and my email I am receiving the same error. The error is as follows 

```
"code: AUTH",
"response": "535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8 https://support.google.com/mail/?p=BadCredentials PASSWORD - gsmtp",
"responseCode": 535,
"command": "AUTH PLAIN",
"stack": "Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8 https://support.google.com/mail/?p=BadCredentials PASSWORD - gsmtp\n at SMTPConnection._formatError (C:\Users\I will get a 3000\Desktop\streetmerchant\node_modules\nodemailer\lib\smtp-connection\index.js:774:19)\n at SMTPConnection._actionAUTHComplete (C:\Users\I will get a 3000\Desktop\streetmerchant\node_modules\nodemailer\lib\smtp-connection\index.js:1513:34)\n at SMTPConnection. (C:\Users\I will get a 3000\Desktop\streetmerchant\node_modules\nodemailer\lib\smtp-connection\index.js:540:26)\n at SMTPConnection._processResponse (C:\Users\I will get a 3000\Desktop\streetmerchant\node_modules\nodemailer\lib\smtp-connection\index.js:932:20)\n at SMTPConnection._onData (C:\Users\I will get a 3000\Desktop\streetmerchant\node_modules\nodemailer\lib\smtp-connection\index.js:739:14)\n at TLSSocket.SMTPConnection._onSocketData (C:\Users\I will get a 3000\Desktop\streetmerchant\node_modules\nodemailer\lib\smtp-connection\index.js:189:44)\n at TLSSocket.emit (node:events:376:20)\n at addChunk (node:internal/streams/readable:304:12)\n at readableAddChunk (node:internal/streams/readable:279:9)\n at TLSSocket.Readable.push (node:internal/streams/readable:218:10)"
```

This error is both the same for my phone number and for my email, yes it is a gmail. None the less I know there have been a few other threads on this none of which have fixed my problem or quite frankly have had any explanation what so ever. Although I have been getting Pushover notifications email and phone are going to be the most viable for me. If I could get help for this issue it would be greatly appreciated, good luck getting your cards, cpu's or consoles!

**Comment by jef** [comment]
Unfortunately, I can't reproduce this. Anytime I setup email, it doesn't seem to be a problem. Are you using an [app password](https://myaccount.google.com/apppasswords)?

Can you post your `.env`? While redacting any secrets.

**Comment by TheRealMystical** [comment]
> Unfortunately, I can't reproduce this. Anytime I setup email, it doesn't seem to be a problem. Are you using an [app password](https://myaccount.google.com/apppasswords)?
> 
> Can you post your `.env`? While redacting any secrets.

COUNTRY="usa"
EMAIL_PASSWORD=""
EMAIL_TO="xxxxxxxx@gmail.com"
EMAIL_USERNAME="xxxxxxxx@gmail.com"
HEADLESS="true"
IN_STOCK_WAIT_TIME="0"
LOG_LEVEL="info"
LOW_BANDWITH="false"
NVIDIA_ADD_TO_CART_ATTEMPTS="10"
NVIDIA_SESSION_TTL="60000"
OPEN_BROWSER="true"
PAGE_BACKOFF_MIN="10000"
PAGE_BACKOFF_MAX="3600000"
PAGE_SLEEP_MIN="5000"
PAGE_SLEEP_MAX=10000""
PAGE_TIMEOUT="0"
PHONE_CARRIER="att"
PHONE_NUMBER="9xxxxxxxxx"
PUSHOVER_TOKEN=""
PUSHOVER_USER=""
SCREENSHOT="true"
SHOW_ONLY_BRANDS="founders edition,rog strix,evga,gigabyte,aorus,msi,asus"
SHOW_ONLY_SERIES="3080,3090"
STORES="newegg"

Here is my .env also I should probably mention I am on a virtualbox vm but I'm not sure if that has anything to do with it, also it says something on the code about enabling smpt, could that be my problem? Also hopefully you didn't fully close this cause it's definitely not resolved. Anyways thank you jef

**Comment by TheRealMystical** [comment]
also I am not using an app password

**Comment by TheRealMystical** [comment]
Alright lets close this out once and for all, because I did some troubleshooting of my own and got it working. First step to my troubleshooting was making a throwaway email, once that was created I allowed less secure apps to be put on, ran the test and it worked I also make sure pop and IMAP are enabled within your email account so that it can be sent across servers. The whole time I thought it was my env settings but it really was my gmail settings. If enabling pop and Imap doesn't work make sure you fiddle with your gmail settings because it probably has something to do with that.
